### PR TITLE
Publicly import `ConfigException` from `Config`, update example

### PR DESCRIPTION
**Pull request by Geod24**
Fairly straightforward.

### CODE REVIEW

**Review by hewison-chris — APPROVED** [review]
LGTM

File: examples/helloworld/source/app.d [hunk 1]
```diff
@@ -31,16 +31,35 @@ void main (string[] args)
...
-    auto config = clargs.parseConfigFile!Config;
+    try
+    {
+        // An `Exception` will be thrown if this fails
+        auto config = clargs.parseConfigFile!Config;
 
-    for (size_t i = 0; config.limit == 0 || i < config.limit; ++i)
+        for (size_t i = 0; config.limit == 0 || i < config.limit; ++i)
+        {
+            writeln("Hello World to you ", config.name);
```
Comment (linked0) [inline examples/helloworld/source/app.d:44]: Doesn't it print the same `config.name` in the for loop?

Comment (Geod24) [reply examples/helloworld/source/app.d:44]: It does, why ?

Comment (linked0) [reply examples/helloworld/source/app.d:44]: Ah, I thought the `for` loop prints the same thing. there's something I need to know about `Config`. Let me check the class more.

Comment (Geod24) [reply examples/helloworld/source/app.d:44]: This code is just an example, so it doesn't do anything really logical. The main point is to show how a user would parse a config file.

**Pull request merged**